README.md

```diff
@@ -0,0 +1,34 @@
+1. Consider the type of data we will be storing and therefore the type of database we should implement (SQL vs NoSQL)
+
+- Relational database with SQL
+
+2. Create a schema for this database
+
+
+
+3. Consider the requests our API should be capable of handling
+
+- Store people, houses and addresses
+- Look up a house, it’s address and owner
+- Look up people in our neighbourhood within certain age brackets and with specific household sizes
+
+4. List the routes you will need with their HTTP verb and path
```

Comment:
Could you explain choices of HTTP verbs, for example a GET where you appear to be editing (PATCH might be more suitable?)
Also there is no POST verb so how will users add data to the database?